services/wfm/src/test/java/org/openkilda/wfm/topology/switchmanager/service/impl/ValidationServiceImplTest.java

```diff
@@ -210,11 +245,28 @@ private PersistenceManager build() {
             when(flowPathA.getCookie()).thenReturn(new Cookie(1));
             when(flowPathA.getMeterId()).thenReturn(new MeterId(32L));
 
-            Flow flow = mock(Flow.class);
-            when(flow.getFlowId()).thenReturn("test_flow");
-            when(flowPathA.getFlow()).thenReturn(flow);
+            Flow flowA = mock(Flow.class);
+            when(flowA.getFlowId()).thenReturn("test_flow");
+            when(flowPathA.getFlow()).thenReturn(flowA);
+
+            Switch switchE = Switch.builder()
+                    .switchId(SWITCH_ID_B)
```

Comment:
`SWITCH_ID_E` ?

Reply:
Now I'm not sure how these tests pass considering the ID is wrong

Reply:
We don't get switch by ID. And when we call `flowPathB.getSrcSwitch()`, we get switchE with "E" manufacturer (when(flowPathB.getSrcSwitch()).thenReturn(switchE)). But I've fixed this mistake.